app/controllers/vulneruby_engine/insecure_algorithm_controller.rb

```diff
@@ -4,7 +4,7 @@ module VulnerubyEngine
   # Entry point for the Insecure Hash Algorithm and Random tests
   class InsecureAlgorithmController < ApplicationController
     SOME_HARDCODED_PASSWORD = 'BadPracticeOfStoringPasswordInRepo'
-    SOME_HARDCODED_KEY = 'BadPracticeOfStoringKeyInRepo'
+    SOME_HARDCODED_KEY = [1,2,3]
```

Comment:
Why this change? I'm intrigued :)

Reply:
Well according to our tests in the agent - we may have different values here to trigger hardcode-key so I switched it to trigger it, as the previous one wasn't